wasm.js

```diff
@@ -954,8 +954,37 @@ syscall_fns = {
   },
   142: {
     name: "SYS__newselect",
-    fn: function() {
-      throw "SYS__newselect NYI";
+    fn: function(nfds, readfds_, writefds_, exceptfds_, timeout_) {
+      // ignore exceptfds_
+
+      var timeout = timeout_ / 4;
+      var timeoutSec = heap_uint32[timeout];
+      var timeoutUSec = heap_uint32[timeout + 1];
+      // nfds == 0 means this is just a timeout/delay request
+      if (nfds == 0 && (timeoutSec !== 0 ||  timeoutUSec !== 0)) {
```

Comment:
if nfds != 0 then it is waiting on activity in some FD
we can implement the case where it really is waiting on a FD later...